hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AProxy.java

```diff
@@ -35,11 +36,11 @@
 import static org.apache.hadoop.fs.s3a.S3AUtils.initProxySupport;
 
 /**
- * Unit tests to verify {@link S3AUtils} translates the proxy configurations
+ * Tests to verify {@link S3AUtils} translates the proxy configurations
  * are set correctly to Client configurations which are later used to construct
  * the proxy in AWS SDK.
  */
-public class TestS3AProxy extends AbstractS3ATestBase{
+public class ITestS3AProxy extends AbstractHadoopTestBase {
```

Comment:
sorry, i was confusing you. the old code was implicitly and ITest because it extended S3ATestBase; with the changed superclass it is a unit test, so can be renamed to the original name

Reply:
ah my bad. Should've re-read the comment